core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -460,6 +461,15 @@ public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throw
     mAlluxioHeader = getScheme() + "://" + authority;
     // Set the statistics member. Use mStatistics instead of the parent class's variable.
     mStatistics = statistics;
+
+    AlluxioURI header = new AlluxioURI(mAlluxioHeader);
+    if (header.getAuthority() instanceof UnknownAuthority) {
+      // TODO(zac): In Alluxio 2.0 this warning will be upgraded to an exception
+      LOG.warn("Authority \"{}\" is unknown. The client will not be configured with this authority",
```

Comment:
the user will probably wonder where the authority comes from if not from the URI